torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -285,6 +332,56 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
 
         return model
 
+    def _annotate_conv2d_bn(self, node: Node, operator_spec: Optional[OperatorSpec]) -> None:
+        """
+        Match the following pattern:
+
+          ... -> conv -> bn -> getitem[0] - ...
+
+        Annotate it to get the following pattern after prepare:
+
+                weight -> obs1
+                           |
+          ...  -> obs0 -> conv -> bn -> getitem[0] -> obs2 -> ...
+
+        Note: This is only used for QAT. In PTQ, batchnorm should already be fused into the conv.
+        """
+        if node.op != "call_function" or node.target != operator.getitem or node.args[1] != 0:
+            return
+        getitem_node = node
+        bn_node = getitem_node.args[0]
+        assert isinstance(bn_node, Node)
+        if bn_node.op != "call_function" or bn_node.target != torch.ops.aten._native_batch_norm_legit.default:
+            return
+        conv_node = bn_node.args[0]
+        assert isinstance(conv_node, Node)
+        if conv_node.op != "call_function" or conv_node.target != torch.ops.aten.convolution.default:
+            return
+        if _is_annotated([getitem_node, bn_node, conv_node]):
+            return
+
+        conv_node.meta["target_dtype_info"] = {
+            "input_act_obs_or_fq_ctr": _get_act_obs_or_fq_ctr(operator_spec),
+            "weight_obs_or_fq_ctr": _get_weight_obs_or_fq_ctr(operator_spec),
+            "bias_obs_or_fq_ctr": _get_bias_obs_or_fq_ctr(operator_spec),
+            "output_act_obs_or_fq_ctr": _get_default_obs_or_fq_ctr(),
+            # TODO: validation of weight_index must be set if weight_obs_or_fq_ctr is set
+            "weight_index": 1,
+            # TODO: validation of bias_index must be set if bias_obs_or_fq_ctr is set
+            "bias_index": 2,
+            "_annotated": True,
+        }
+        bn_node.meta["target_dtype_info"] = {
+            "input_act_obs_or_fq_ctr": _get_default_obs_or_fq_ctr(),
+            "output_act_obs_or_fq_ctr": _get_default_obs_or_fq_ctr(),
+            "_annotated": True,
+        }
+        getitem_node.meta["target_dtype_info"] = {
+            "input_act_obs_or_fq_ctr": _get_default_obs_or_fq_ctr(),
```

Comment:
Same here. In fact with this PR, https://github.com/pytorch/pytorch/pull/99071, you can just say None for input.

Reply:
Ok, let's do this later since none of the other annotations are updated to reflect that yet